Multisig/Features/Connect to Wallet/UI/PendingWalletActionViewController.swift

```diff
@@ -26,11 +26,15 @@ class PendingWalletActionViewController: ContainerViewController, UIAdaptivePres
     var requestTimeout: TimeInterval = 120
 
     var walletName: String {
-        wallet?.name ?? connection.remotePeer?.name ?? "wallet"
+        wallet?.name ?? connection?.remotePeer?.name ?? "wallet"
     }
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        if let keyInfo = keyInfo, connection == nil {
+            connection = WebConnectionController.shared.walletConnection(keyInfo: keyInfo).first
```

Comment:
connection will not be overridden if it wasn't nil.  What if keyInfo.connections changes? Or is that impossible?